src/static_api.rs

```diff
@@ -121,6 +135,9 @@ impl<'a> Generator<'a> {
                         github: (*github_name).into(),
                         github_id: person.github_id(),
                         is_lead: leads.contains(github_name),
+                        roles: website_roles
+                            .get(*github_name)
+                            .map_or_else(Vec::new, Vec::clone),
```

Comment:
Nit: this might be slightly more clear:

```suggestion
                            .cloned().unwrap_or_default()
```

Reply:
Fixed.